packages/os/ebs-volumes.rules

```diff
@@ -1,8 +1,10 @@
 ACTION=="remove", GOTO="ebs_volumes_end"
-SUBSYSTEM!="block", GOTO="ebs_volumes_end"
-ENV{DEVTYPE}!="disk", GOTO="ebs_volumes_end"
+
+# Create symlinks for the device name configured in EC2, for both block devices and partitions
+KERNEL=="nvme[0-9]*n[0-9]*", ENV{DEVTYPE}=="disk", ATTRS{model}=="Amazon Elastic Block Store", PROGRAM="/usr/bin/ghostdog ebs-device-name /dev/$kernel", SYMLINK+="$result"
```

Comment:
I'd sort of prefer to do this as part of the `ghostdog scan` operation, but it's OK to have a different subcommand.

I'd like it to take similar arguments though. It should also be set up with similar guards, and run after the io_timeout rule:
```
# add these guards to the top, after ACTION=="remove"
KERNEL!="nvme*", GOTO="ebs_volumes_end"
ATTRS{model}!="Amazon Elastic Block Store", GOTO="ebs_volumes_end"
...

# Set XVD_DEVICE_NAME for disk
KERNEL=="nvme[0-9]*n[0-9]*", ENV{DEVTYPE}=="disk", IMPORT{program}="/usr/bin/ghostdog ebs-device-name $devnode"

# Set XVD_DEVICE_NAME for partition
KERNEL=="nvme[0-9]*n[0-9]*p[0-9]*", ENV{DEVTYPE}=="partition", IMPORT{parent}="{XVD_DEVICE_NAME}"

# Use XVD_DEVICE_NAME
XVD_DEVICE_NAME=="xvd*", SYMLINK+="$env{XVD_DEVICE_NAME}"
```
```